client/layout/sidebar/item.jsx

```diff
@@ -37,7 +38,10 @@ export default React.createClass( {
 
 	render() {
 		const isExternalLink = isExternal( this.props.link );
-		const classes = classnames( this.props.className, { selected: this.props.selected } );
+		const classes = classnames( 'sidebar__item', this.props.className, {
```

Comment:
Minor and optional but since this `render()` method uses quite a few props (and some more then once like `link`) it might be worth destructuring above so `this.props` doesn't need repeated so many times.